wallet/how-to/connect/set-up-sdk/gaming/unity.md

```diff
@@ -190,6 +192,20 @@ Create a new configuration by right-clicking on the project window and navigatin
 **MetaMask > Config**.
 Name the new configuration and use it when initializing the `MetaMaskUnity` instance.
 
+## Update the Unity SDK
+
+Update your project's version of the MetaMask Unity SDK to ensure that your project uses the SDK's
+latest functionalities and improvements.
```

Comment:
uses the SDK's latest functionalities and improvements. = uses the latest SDK functionalities and improvements.